src/map.cpp

```diff
@@ -1218,7 +1226,12 @@ ServerMap::ServerMap(const std::string &savedir, IGameDef *gamedef,
 	m_savedir = savedir;
 	m_map_saving_enabled = false;
 
-	m_save_time_counter = mb->addCounter("minetest_core_map_save_time", "Map save time (in nanoseconds)");
+	m_save_time_counter = mb->addCounter(
+		"minetest_map_save_time", "Time spent saving blocks (in microseconds)");
```

Comment:
Should probably use a consistent naming pattern: `minetest_core_`. Same applies to other added metrics.

The reason to prefix by `minetest_core_` is to distinguish between engine metrics and mod metrics (which may be added by another prometheus service or by a future API)

Reply:
> Note: This renames minetest_core_map_save_time to minetest_map_save_time and changes the unit to microseconds.

Ah...   Idk then

Reply:
The naming scheme here is: minetest_&lt;area>_&lt;thing>
where &lt;area> is one of core (server.cpp), env (serverenvironment.cpp), map (map.cpp), emerge (emerge*.cpp)

When mod metrics are added I think it's fine to let mods name their stuff whatever they want but give advice to use `minetest_mod_<modname>_<thing>`

Reply:
OK fair enough